docs/components/best-practices/architecture/deciding-about-your-stack-c7.md

```diff
@@ -94,7 +94,7 @@ Check the **prerequisites**:
 
 Create your **development project**
 
-1. Create a new Spring Boot project (e.g. using [Spring initializr](https://start.spring.io/))
+1. Create a new Spring Boot project (e.g. using [Spring initializer](https://start.spring.io/))
```

Comment:
This is a product/project name, so the previous spelling is actually correct! We should capitalize "Initializr" though - https://github.com/spring-io/initializr

Reply:
The technical writer in me is crying without the "e", but this has been adjusted 👍